orca-clouddriver/src/test/java/com/netflix/spinnaker/orca/clouddriver/tasks/monitoreddeploy/MonitoredDeployBaseTaskTest.java

```diff
@@ -168,6 +173,99 @@ void shouldReturnDefaultLogMsgWhenUnexpectedErrorHasOccurred() {
     assertThat(logMessageOnUnexpectedError).isEqualTo("<NO RESPONSE>");
   }
 
+  @Test
+  void shouldReturnEmptyHttpErrorDetailsWhenExceptionThrownWhileReadingHttpStatus() {
```

Comment:
Seems like there's a much simpler way to test this case, e.g.:
```
  @Test
  void returnsEmptyHttpErrorDetailsWhenExceptionThrownWhileReadingHttpStatus() {

    Response response = mock(Response.class);
    when(response.getStatus()).thenThrow(IllegalArgumentException.class);

    String logMessageOnHttpError = monitoredDeployBaseTask.getRetrofitLogMessage(response);

    assertThat(logMessageOnHttpError).isEqualTo("status: \nheaders: \nresponse body: ");
  }
```

Reply:
Simplified the test case that verifies the scenario when getRetrofitLogMessage() returns empty http error details.